app/Providers/AppServiceProvider.php

```diff
@@ -101,6 +102,14 @@ protected function configureAuraServices(\Aura\Di\ContainerInterface $di)
             'clientUrl' => $di->lazyGet('clienturl')
         ]));
 
+        // Configure filesystem
+        // The Ushahidi filesystem adapter returns a flysystem adapter for a given
+        // cdn type based on the provided configuration
+        $di->set('tool.filesystem', function () {
+            // Get the underlying League\Flysystem\Filesystem instance
+            return $this->app->make('filesystem')->disk()->getDriver();
```

Comment:
Future music: We should define both public and private filesystem. CSV uploads could be private, but image uploads should be public.